spec/unit/services_inspector_spec.rb

```diff
@@ -71,11 +78,14 @@
       expect(summary).to eq("Found 3 services.")
     end
 
-    it "returns data about SysVinit services when no systemd is present" do
+    it "returns data about SysVinit services on a suse system when no systemd is present" do
       system = double
       expect(system).to receive(:run_command).
         with("systemctl", "--version").
         and_raise(Cheetah::ExecutionFailed.new(nil, nil, nil, nil))
+      expect(system).to receive(:run_command).
+        with("chkconfig", "--version").
+        and_raise(Cheetah::ExecutionFailed.new(nil, nil, nil, nil))
```

Comment:
It's not very nice that this tests has to be adapted, when the code change actually is not related to what is tested by this test. Is there a better way?